api/mysagw/case/serializers.py

```diff
@@ -6,6 +6,10 @@
 
 
 class CaseAccessSerializer(serializers.ModelSerializer):
+    included_serializers = {
+        "identity": "mysagw.identity.serializers.IdentitySerializer",
```

Comment:
With this I think we'll face the same problem as with the profile view and give too many infos to non-admin users. I think we should add a custom serializer only providing
 - email
 - idp_id
 - first_name
 - last_name

Reply:
Added `PublicIdentitySeriazlizer`